### PR TITLE
Add a default maven_server option

### PR DESCRIPTION
**Pull request by GinFungYJF**

### CODE REVIEW

File: src/main/java/braintree/BazelDeps.java [hunk 1]
```diff
@@ -24,6 +24,9 @@
   @Option(name = "-x", usage = "Exclude a libraries dependencies")
   private String excludeArtifact;
 
+  @Option(name = "-r", usage = "specify a default maven server")
```
Comment (pgr0ss) [inline src/main/java/braintree/BazelDeps.java]: Can you capitalize `specify`?

File: src/main/java/braintree/BazelDeps.java [hunk 2]
```diff
@@ -69,6 +72,9 @@ public void doMain(String[] args) throws DependencyCollectionException, CmdLineE
   private void printWorkspace(Map<Artifact, Set<Artifact>> dependencies,
                               Set<Artifact> excludeDependencies) {
     System.out.println("\n\n--------- Add these lines to your WORKSPACE file ---------\n");
+    if(repository !=null){
```
Comment (pgr0ss) [inline src/main/java/braintree/BazelDeps.java]: Can you fix the spacing here?

**Review by pgr0ss — APPROVED** [review]
Super minor feedback and then I'll merge it. Thanks!

**Comment by GinFungYJF** [comment]
Fixed. Can you review again, thanks.

**Pull request merged**